jobserver/models/core.py

```diff
@@ -328,6 +328,13 @@ def status(self):
 class Org(models.Model):
     """An Organisation using the platform"""
 
+    created_by = models.ForeignKey(
+        "User",
+        null=True,
+        on_delete=models.SET_NULL,
```

Comment:
From memory, this means that if the instance of `User` is deleted, then this value is set to null. Under what circumstances do we delete users?

Reply:
That's correct.  We're unlikely to delete users I think, but as of a recent version (3 maybe?) Django requires `on_delete` to be specified and this seemed like the logical choice for this field.

Reply:
I'm in two minds. On the one hand, this isn't really the best place for enforcing a policy ("Users must not be deleted"). On the other hand, if we want to track which users created which organizations and projects over any length of time, then we should make it hard to delete users. Might this be worth a broader discussion? We could, for example, consider `models.PROTECT`.

Reply:
We don't currently have this data for any of the existing Projects or Orgs so I'm also mirroring the existing situation.

But you make a very valid point, we could definitely bring it up in the weekly meeting.

Note one currently has to go into the database to delete users (where this constraint isn't enforced) so it's rather moot right now!

Reply:
Very reasonable: we don't want to arbitrarily assign a user to existing projects or organizations, so the DB is modelling current use 🙂 If we have time, then it would be good to discuss in the weekly meeting. I'll add it to the agenda.